float64 back_lbz
float64 neck_ay
---
int32 success

